src/desdemona/query.clj

```diff
@@ -43,9 +43,13 @@
     [((= value ((attr lvar) :seq)) :seq)]
     `(l/featurec ~lvar {~attr ~value})
 
-    [(['and & terms] :seq)]
+    [(('and & terms) :seq)]
```

Comment:
This changes nothing about the semantics; it simply makes it more consistent with above code, and makes it look more like the form it's actually evaluating.